pysatNASA/instruments/ses1_gold.py

```diff
@@ -0,0 +1,190 @@
+"""Supports the Nmax data product from the Global Observations of the Limb and
+Disk (GOLD) satellite.  Accesses data in netCDF format.
+
+Properties
+----------
+platform
+    'ses1'
+name
+    'gold'
+tag
+    'nmax'
+
+Warnings
+--------
+- The cleaning parameters for the instrument are still under development.
+- strict_time_flag must be set to False
+
+
+Examples
+--------
+::
+
+    import datetime as dt
+    import pysat
+    nmax = pysat.Instrument(platform='ses1', name='gold', tag='nmax'
+                            strict_time_flag=False)
+    nmax.download(dt.datetime(2020, 1, 1), dt.datetime(2020, 1, 31))
+    nmax.load(2020, 1)
+
+
+Authors
+---------
+Jeff Klenzing, Oct 06, 2020, Goddard Space Flight Center
+
+"""
+
+import datetime as dt
+import functools
+import warnings
+
+from pysat import logger
+from pysat.instruments.methods import general as ps_gen
+from pysat.utils import load_netcdf4
+from pysatNASA.instruments.methods import gold as mm_gold
+from pysatNASA.instruments.methods import cdaweb as cdw
+
+# ----------------------------------------------------------------------------
+# Instrument attributes
+
+platform = 'ses1'
+name = 'gold'
+tags = {'nmax': 'Nmax data for the GOLD instrument'}
+inst_ids = {'': ['nmax']}
+
+pandas_format = False
+
+# ----------------------------------------------------------------------------
+# Instrument test attributes
+
+_test_dates = {'': {'nmax': dt.datetime(2020, 1, 1)}}
+
+# ----------------------------------------------------------------------------
+# Instrument methods
+
+
+def init(self):
+    """Initializes the Instrument object with instrument specific values.
+
+    Runs once upon instantiation.
+
+    Parameters
+    -----------
+    self : pysat.Instrument
+        Instrument class object
+
+    """
+
+    logger.info(mm_gold.ack_str)
+    logger.warning(' '.join(('Time stamps may be non-unique because Channel A',
+                             'and B are different instruments.  An upgrade to',
+                             'the pysat.Constellation object is required to',
+                             'solve this issue. See pysat issue #614 for more',
+                             'info.')))
+    self.acknowledgements = mm_gold.ack_str
+    self.references = mm_gold.ref_str
+
+    pass
```

Comment:
```suggestion
    return
```

Because this is the end of the method.